src/usr/local/sbin/pfSense-upgrade

```diff
@@ -860,3 +864,4 @@ case "${action}" in
 esac
 
 _exit 0
+\
```

Comment:
Oops - I leaned on the keyboard. I will make a new PR, so there is not a silly little commit that reverses this.